pkg/login/auth.go

```diff
@@ -63,10 +65,6 @@ func AuthenticateUser(query *models.LoginUserQuery) error {
 		return ErrInvalidCredentials
 	}
 
-	if err == models.ErrUserNotFound {
-		return ErrInvalidCredentials
```

Comment:
Random thought: If we'd have a custom `AuthenticationError` type we would be able to send metadata about what's wrong that isn't part of the `.Error()` 🤔 That would allow for making sure that any and all authentication errors give a vague message outwards while retaining full legibility internally.